build.gradle

```diff
@@ -46,13 +46,13 @@ repositories {
 
 dependencies {
     testCompile group: 'junit', name: 'junit', version: '[4.3,5.0['
-    testCompile group: 'com.github.tomakehurst', name: 'wiremock', version: '[1.45,1.99999)'
-    compile group: 'commons-lang', name: 'commons-lang', version: '[2.6,2.99999)'
+    testCompile group: 'com.github.tomakehurst', name: 'wiremock', version: '[1.45,2.0['
```

Comment:
can we use `)` instead of `[` to end these - that is some pretty unclear syntax 😛

Reply:
Swapped to `]` so that it's inclusive (I was adding support for 2.0)